src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
Why use `onclick` and a handler rather than using `react-router`'s [`goBack()`](https://github.com/ReactTraining/react-router/blob/master/docs/guides/NavigatingOutsideOfComponents.md)?